<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(126, 116, 185);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="txt_1">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Hola</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_accion">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 23, 23);
border-color: rgb(0, 0, 0);
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Pasar informaci√≥n</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 167, 25);
border-color: rgb(53, 0, 127);
border: 2px  solid  rgb(0, 170, 255);
border-style: outset;
border-width: 2px;
border-radius: 20px;</string>
      </property>
      <property name="text">
       <string>e</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
